demos/browser/package.json

```diff
@@ -26,15 +26,15 @@
     "sass": "^1.26.11",
     "sass-loader": "^10.0.2",
     "style-loader": "^1.2.1",
-    "ts-loader": "^8.0.4",
-    "typescript": "^4.0.0",
+    "ts-loader": "^8.1.0",
+    "typescript": "^4.2.3",
     "webpack": "^4.44.2",
-    "webpack-cli": "^3.3.12",
+    "webpack-cli": "^4.6.0",
```

Comment:
Does anything change here due to the major version upgrade? I checked [here](https://webpack.js.org/migrate/4/#cli) but looks like we should be good.

Reply:
I assume we have verified that npm run start still works after this :D

Reply:
Correct :D